source/guide_simpleid.rst

```diff
@@ -0,0 +1,209 @@
+
+  * PHP_
+  * domains_
+
+Prerequisites
+=============
+
+We're using PHP_ in the stable version 7.1:
+
+::
+
+ [isabell@stardust ~]$ uberspace tools version show php
+ Using 'PHP' version: '7.1'
+ [isabell@stardust ~]$
+
+Your URL needs to be setup for web:
+
+::
+
+ [isabell@stardust ~]$ uberspace web domain list
+ isabell.uber.space
+ [isabell@stardust ~]$
+ 
+Installation
+============
+
+Step 1 - Download & Extract
+------------------------------
+
+``cd`` to your `document root`_, respectively the folder above, because not all files must/should be accessable via web, then download the latest release of *SimpleID* and extract it:
+
+.. note:: The link to the lastest version can be found at SimpleID's `download page <http://simpleid.koinic.net/releases//>`_.
+
+::
+
+ [isabell@stardust ~]$ cd /var/www/virtual/$USER/
+ [isabell@stardust isabell]$ wget http://downloads.sourceforge.net/simpleid/simpleid-1.0.2.tar.gz
+ [isabell@stardust isabell]$ tar -xzf simpleid-1.0.2.tar.gz
+
+Step 2 - Symlink
+----------------
+
+The folder containing the frontend needs to be accessible via web. To achieve this, a symlink is created linking to the www folder.
+
+::
+
+ [isabell@stardust isabell]$ ln -s /var/www/virtual/$USER/simpleid/www/ ~/html/simpleid
+ 
+Step 3 - Copy empty configuration
+---------------------------------
+
+::
+
+ [isabell@stardust isabell]$ cp /var/www/virtual/$USER/simpleid/www/config.php.dist /var/www/virtual/$USER/simpleid/www/config.php
+
+Step 4 - Cleanup
+----------------
+::
+
+ [isabell@stardust isabell]$ rm simpleid-1.0.2.tar.gz
+ 
+Configuration
+=============
+
+Define Base URL
+---------------
+
+Edit file ``/var/www/virtual/$USER/simpleid/www/config.php``.
+The only change you really have to do is changing the SIMPLEID_BASE_URL, the other options are optional.
+
+.. warning:: Replace ``<username>`` with your Uberspace username!
+
+.. code-block:: php
+
+ define('SIMPLEID_BASE_URL', 'https://<username>.uber.space/simpleid');
+ 
+In our example this would be:
+ 
+.. code-block:: php
+
+ define('SIMPLEID_BASE_URL', 'https://isabell.uber.space/simpleid');
+
+Create identity
+---------------
+
+You will need to create an identify file for every user of your SimpleID installation.
+
+.. warning:: Replace ``<username>`` with the username you want to use for your SimpleID identity!
+
+::
+
+ [isabell@stardust isabell]$ cp /var/www/virtual/$USER/simpleid/identities/example.identity.dist /var/www/virtual/$USER/simpleid/identities/<username>.identity 
+
+Before editing this file, we need to create a password/salt pair.
+
+There are several ways to go, using the MD5, SHA1 and SHA256 algorithm and an optional salt. In this example we will use the SHA256 algorithm with a random salt.
+
+First we'll generate a random salt to make this secure as possible. You are free to use any string as salt.
+
+::
+
+ [isabell@stardust ~]$ date +%s | sha256sum | base64 | head -c 16 ; echo
+ Y2MyZTNkYzI0OTA3
```

Comment:
Please change this to "SuperSecretSalt" or something like that. Users will just use this example otherwise.